scripts/content/VACMS-6456-remove-leading-care-we-provide-h3-tags.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * @file
+ * Remove leading Care we provide at [location] h3 tags.
+ *
+ *  VACMS-6456-remove-leading-care-we-provide-h3-tags.php.
+ */
+
+use Psr\Log\LogLevel;
+
+$sandbox = ['#finished' => 0];
+do {
+  print(va_gov_remove_health_service_node_headings($sandbox));
+} while ($sandbox['#finished'] < 1);
+// Node processing complete.  Call this done.
+return;
+
+/**
+ * Remove headings from VAMC System Health Service nodes matching RegEx.
+ *
+ * @param array $sandbox
+ *   Modeling the structure of hook_update_n sandbox.
+ *
+ * @return string
+ *   Status message.
+ */
+function va_gov_remove_health_service_node_headings(array &$sandbox) {
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  // Get the node count for VAMC System Health Service nodes.
+  // This runs only once.
+  if (!isset($sandbox['total'])) {
+    $query = Drupal::database()->select('node_field_data', 'nfd');
+    $query->join('node__field_body', 'nfb', 'nfd.nid = nfb.entity_id');
+    $query->fields('nfd', ['nid']);
+    $query->condition('nfd.type', 'regional_health_care_service_des');
+    $query->condition('nfb.field_body_value', '^<h3>Care we provide at .+ health care</h3>', 'REGEXP');
+    $nids_to_update = $query->execute()->fetchCol();
+    $result_count = count($nids_to_update);
+    $sandbox['total'] = $result_count;
+    $sandbox['current'] = 0;
+    // Create non-numeric keys to accurately remove each nid when processed.
+    $sandbox['nids_to_update'] = array_combine(
+      array_map('_va_gov_stringifynid', array_values($nids_to_update)),
+      array_values($nids_to_update));
+  }
+
+  // Do not continue if no nodes are found.
+  if (empty($sandbox['total'])) {
+    $sandbox['#finished'] = 1;
+    return "No vamc system health service nodes were found to be processed.\n";
+  }
+
+  $limit = 25;
+
+  // Load entities.
+  $node_ids = array_slice($sandbox['nids_to_update'], 0, $limit, TRUE);
+  $nodes = $node_storage->loadMultiple($node_ids);
+
+  foreach ($nodes as $node) {
+    // Remove leading <h3> tag.
+    $field_body = $node->get('field_body')->value;
+    $new_body = preg_replace('/<[hH]3>[cC]are [wW]e [pP]rovide at .+ [hH]ealth [cC]are<\/[hH]3>/', '', $field_body, 1);
+
+    // Compare old and new node body and make sure its updated before saving.
+    if ($field_body !== $new_body) {
+      // Make this change a new revision.
+      $node->setNewRevision(TRUE);
+
+      // Update field_body with the new value.
+      $node->field_body->setValue(
+        [
+          'value' => $new_body,
+          'format' => 'rich_text',
+        ]
+      );
+
+      // Set revision author to uid 1317 (CMS Migrator user).
+      $node->setRevisionUserId(1317);
+      $node->setChangedTime(time());
+      $node->setRevisionCreationTime(time());
+
+      // Set revision log message.
+      $node->setRevisionLogMessage('Resaved node with leading h3 removed.');
+
+      // Save the changes.
```

Comment:
```suggestion
      // Save the changes.
```
Minor: No clearer than the code itself